docs/tutorials/parameter-sweeping.md

```
@@ -0,0 +1,944 @@
+```
+Minimal output, fastest execution - ideal for automated runs or CI/CD pipelines.
+
+### Dashboard UI Not Supported
+
+The dashboard UI (`--ui dashboard`) is incompatible with parameter sweep mode. If you explicitly try to use it, you'll get an error:
+
+```bash
+aiperf profile --concurrency 10,20,30,40 --ui dashboard ...
+```
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)